cfg/gc/run-excalibur.sh

```diff
@@ -29,18 +29,18 @@ cd -
 
 # Run analysis with config from worker node tmp dir
 echo "#### Starting Artus..."
-_TME_FILE='artus.tme'
+_TME_FILE='excalibur.tme'
 if [[ $(which time) ]]; then
 	# GNU time allows for extended format - use IEEE POSIX.2 as base
-	$(which time) -o ${_TME_FILE} -f 'real %e\nuser %U\nsys %S\nrss %M\nmajor_pfault %F\nmajor_pfault%R\nwaits %w' @WORKPATH@/artus @ARTUS_CONFIG@ || exit 1
+	$(which time) -o ${_TME_FILE} -f 'real %e\nuser %U\nsys %S\nrss %M\nmajor_pfault %F\nmajor_pfault%R\nwaits %w' excalibur @ARTUS_CONFIG@ || exit 1
```

Comment:
Thanks Max for your reply, 

nice think about scram compile is that it is automatically in the $PATH when you set the cmsenv ;)

And for the major issue, we are planing to switch from USER mode to an CMSSW task, which automatically recovers all off the functionality you described. This changes are only made the get a working version of excalibur.